types/src/system/auction/entry_points.rs

```diff
@@ -76,6 +77,20 @@ pub fn auction_entry_points() -> EntryPoints {
     );
     entry_points.add_entry_point(entry_point);
 
+    let entry_point = EntryPoint::new(
+        METHOD_REDELEGATE,
+        vec![
+            Parameter::new(ARG_DELEGATOR, PublicKey::cl_type()),
+            Parameter::new(ARG_VALIDATOR, PublicKey::cl_type()),
+            Parameter::new(ARG_AMOUNT, U512::cl_type()),
+            Parameter::new(ARG_NEW_VALIDATOR, Option::<PublicKey>::cl_type()),
```

Comment:
if someone calles `redelegate`, shouldn't the `ARG_NEW_VALIDATOR` argument be required?

Reply:
It is required, and has been changed to reflect that